test_project/settings.py

```diff
@@ -79,7 +79,9 @@
     'django.contrib.sites',
     'django.contrib.admin',
     'django.contrib.messages',
-    
+
     'jsonate',
     'test_app',
 )
+
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
```

Comment:
Ah, this one should actually be `DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'` as per MP's [#2736](https://github.com/mediapredict/mediapredict/issues/2736#issue-981507834).
This way the only migration we'll do will be the one regarding BooleanField

Reply:
ohhhh...I thought Django was smart enough to only apply that value to new tables.  I'm thinking of some other projects where that migration would have been a disaster.